scripts/qa_crowbarsetup.sh

```diff
@@ -2341,6 +2341,16 @@ function addfloatingip()
     nova add-floating-ip "$instanceid" "$floatingip"
 }
 
+# by setting --dns-nameserver for subnet, docker instance gets this as
+# DNS info (otherwise it would use /etc/resolv.conf from its host)
+function adapt_dns_for_docker()
+{
+    # DNS server is the first IP from the allocation pool, or the
+    # second one from the network range
+    local dns_server=`neutron subnet-show fixed | grep allocation_pools | cut -d '"' -f4`
```

Comment:
related to what i asked in https://github.com/SUSE-Cloud/automation/pull/439 can we rely on `neutron subnet-show fixed` to return something? is this subnet created somewhere in mkcloud before we reach this code?